components/dashboard/src/settings/selectClass.tsx

```diff
@@ -18,12 +19,10 @@ interface SelectWorkspaceClassProps {
 export default function SelectWorkspaceClass(props: SelectWorkspaceClassProps) {
     const { user } = useContext(UserContext);
 
-    const [workspaceClass, setWorkspaceClass] = useState<string>(
-        user?.additionalData?.workspaceClasses?.regular || "g1-standard",
-    );
+    const [workspaceClass, setWorkspaceClass] = useState<string>(user?.additionalData?.workspaceClasses?.regular || "");
     const actuallySetWorkspaceClass = async (value: string) => {
         const additionalData = user?.additionalData || {};
-        const prevWorkspaceClass = additionalData?.workspaceClasses?.regular || "g1-standard";
+        const prevWorkspaceClass = additionalData?.workspaceClasses?.regular || "";
```

Comment:
nit: The preview env classes read "Standard" and "Default", where Default is the bigger one which was somewhat confusing.

:bulb: We could re-use the SaaS names (+ maybe sizes?).

Reply:
Ah, that is only because I edited the configmap to better illustrate what it would look like. The installer only installs the Default workspace class as Tarun has posted below.